README.md

```diff
@@ -1,57 +1,82 @@
 # Introduction
 
-### NB!!! Make sure you're checked out on the develop branch
-
 This system is an employee management system for Retro Rabbit Enterprise Services. This is the back end for said system and works in conjuncture with the Front End repository
 
+# Prerequisites
+- [Visual Studio Community 2022](https://visualstudio.microsoft.com/vs/community/)
+- [PgAdmin 4 (V8.6 or above)](https://www.pgadmin.org/download/pgadmin-4-windows/)
+- [Node.js (v20.13.1 or above)](https://nodejs.org/en/download/prebuilt-installer)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [Git (v2.45.1 or above)](https://git-scm.com/download/win)
+- [Docker Desktop (Or any other git UI application)](https://desktop.github.com/)
+
 # Getting Started
 
-### Cloning the [repository](https://retro-rabbit@dev.azure.com/retro-rabbit/RetroGradOnboard/_git/RGO-Server)
+## Docker
+
+Docker is used to run PostgreSql.
+
+### Installing Docker
+
+- If you don't use **WSL/Ubuntu** subsystem, install Docker using the **Hyper-V** installation
+- If this is a new installation, follow the default settings for install
+
+- If you get a `WSL wrong version error` run the following command 
 
 ```powershell
-git clone 'https://retro-rabbit@dev.azure.com/retro-rabbit/RetroGradOnboard/_git/RGO-Server'
+wsl --install
 ```
 
-Runs on(.NET Web API):
+- Run the below command to install Postgres container in Docker
+```powershell
+docker run --name RGO -e POSTGRES_PASSWORD=postgrespw -p 5432:5432 -d postgres
+```
 
-- https://localhost:7026
-- http://localhost:5193
+## Environment Variables
+You need to add environment variables onto your machine.
 
-## Docker
+| Variable name                      | Variable value |
+| --------------------------         | -------------- |
+| Auth__Audience                     | RGO Client     |
+| Auth__Expires                      | 60             |
+| Auth__Issuer                       | RGO API        |
+| Auth__Key                          | ########       |
+| ConnectionStrings__Default         | ########       |
+| NewEmployeeQueue__ConnectionString | ########       |
```

Comment:
Once the environment variables have been created/saved, Visual Studio needs to be restarted.